internal/clients/http/sources/sources_client.go

```diff
@@ -49,15 +49,6 @@ func newSourcesClient(ctx context.Context) (clients.Sources, error) {
 	return &sourcesClient{client: c, logger: logger}, nil
 }
 
-func copySource(src Source) clients.Source {
```

Comment:
it looked bit more clean to me to have this separate, why did you inlined it? any specific reason?

Reply:
Because it's not used anywhere else, we do a ton of copying in other places and we do it without helper functions.